README.md

```diff
@@ -7,122 +7,86 @@
 
 CLI for the Griptape Framework and Cloud.
 
-## Prerequisites
-
-CLI `cloud` commands require a [Griptape Cloud](https://www.griptape.ai/griptape-cloud/) account.
-
-## Quick Start
+## Installation 
 
 1. Install griptape-cli
-   ```
+   ```bash
    pipx install griptape-cli
    ```
-1. Verify installation
+
+   You can also install a pre-release version from GitHub.
+   ```bash
+   git clone https://github.com/griptape-ai/griptape-cli.git
    ```
+
+   ```bash
+   pipx install ./griptape-cli
+   ```
+2. Verify installation
+   ```bash
    gt --help
    ```
 
-## Griptape Cloud
 
-### Skatepark Emulator
+## Skatepark Emulator
 You can use the CLI to spin up a local emulator of the Griptape Cloud Skatepark. This is useful for testing and development.
 
-1. Start by creating a new directory and navigating to it.
-```
-mkdir skatepark-local 
-cd skatepark-local
-```
+1. Start by creating a new repository from the [Managed Structure Template](https://github.com/griptape-ai/managed-structure-template).
```

Comment:
Can you include the steps they'll go through to create a new repo from this one?

* Make sure you're logged in to GitHub
* Go to the [Managed Structure](https://github.com/griptape-ai/managed-structure-template) Template repo
* Click **Use this template**
* Choose **Create a new repository**